roles/docker/tasks/cleanup.yml

```diff
@@ -12,3 +12,9 @@
   become: true
   with_items:
     - "{{ docker_api_port }}"
+
+- name: cleanup contiv-compose
```

Comment:
yes, we already have the version check in place just that looks like we might not be bumping it up when releasing contiv-compose

https://github.com/contiv/ansible/blob/master/roles/docker/tasks/main.yml#L147